canfund-rs/src/manager/record.rs

```diff
@@ -35,6 +39,10 @@ impl CanisterRecord {
     pub fn set_cycles(&mut self, cycles: CyclesBalance) {
         if let Some(previous_cycles) = self.cycles.as_ref() {
             self.previous_cycles = Some(previous_cycles.clone());
+            self.consumtion_history.add_sample(
+                previous_cycles.amount.saturating_sub(cycles.amount) as u64 * 1_000_000_000
```

Comment:
nit: potentially add a comment explaining why the multiplication by `1_000_000_000`